### DISCUSSION TITLE
Bounds.Size = 0 OnLoaded, on Single view platforms (browser & Android)

### DISCUSSION THREAD

**Discussion by Mrcubix** [AvaloniaUI/Avalonia#12850] [Q&A] [answered]
Currently having issue with Avalonia behaving differently on single view platforms like android & wasm.

My app rely on the base size on load to properly scale custom drawn controls, and for some reasons, the bounds of elements are 0 during the `OnLoaded` event on those platforms, no issues on desktop.

Any ideas on how i could get the bounds in a way that works on all platforms?

**Comment by timunie** [comment]
probably it's too slow. In that case, `Dispather.UiThread.Post` can help. Moreover, you may want to listen to `Bounds`-changed events. You can do it in override `OnPropertyChanged`

**Comment by maxkatz6** [comment, marked as answer]
You also can handle SizeChanged event

**Reply by Mrcubix** [reply]
Seems like to work best this way, thanks
